README.md

```diff
@@ -150,25 +150,20 @@ With the Postgres container enabled, users (both students and instructors) can c
 
 ### Spawn Specific Workspace Types
 
-> **New in Version 0.8.0**: users can spawn specific workspace types by setting the `workspace_type` key when initiating the launch request. The `workspace_type` key can either be added as a query parameter or included within the request's body (when initiating an LTI launch).
-
-To enable this feature, you must set the `JupyterHub.spawner_class` to the `IllumiDeskWorkSpaceDockerSpawner` class. Like the `IllumiDeskRoleDockerSpawner` class, the `IllumiDeskWorkSpaceDockerSpawner` class inherits from the `IllumiDeskBaseDockerSpawner` class which manages the logic to assign the image to spawn based on the `workspace_type` key in the `auth_state` dictionary.
-
 Additional workspace types include:
 
 - THEIA IDE: `theia`
 - RStudio: `rstudio`
 - VS Code: `vscode`
 
-If a value is sent which is not recognized by the system, then the default Jupyter Notebook workspace is launched for the user.
 
-When specifying a workspace type with the query parameter add the key / value to the URL encoded request. For example, with LTI 1.1 launch requests the query parameter would look like so:
+When you want to use a specific workspace type you can do it by using the [user redirect](https://jupyterhub.readthedocs.io/en/stable/reference/urls.html#user-redirect) functionality in jupyterhub combined with the query parameter `next`. For example, with LTI 1.1 the launch url would look like so:
 
 ```
-    <url>/lti/launch?next=/user-redirect/theia&workspace_type=theia...
+    <url>/lti/launch?next=/user-redirect/theia
 ```
 
-Notice the `/user-redirect/theia` part. This path redirects the user directly to their user workspace, instead of seeing the default `Launch` button in the application's home page. The path value should correspond with the `workspace_type` value. Users do have the option to navigate back to the Jupyter Notebook interface (Classic or Lab) by appending the `/tree` or `/lab` paths after `.../user/<name>`.
+Notice the `/user-redirect/theia` part. With this the user is redirected to their user workspace, instead of seeing the default `Launch` button in the application's home page. The path value should correspond with the **workspace types** above listed. Users do have the option to navigate back to the Jupyter Notebook interface (Classic or Lab) by appending the `/tree` or `/lab` paths after `.../user/<name>`.
```

Comment:
... Similar to how users can toggle between `/tree` and `/lab` for Jupyter Classic and JupyterLab, respectively, the user may set other workspace types with recognized paths that point to specific workspace types. There is no restriction on what path to use, in so long as the `jupyter-server-proxy` implementation available with the end-user workspace [has that path defined as an entrypoint](https://jupyter-server-proxy.readthedocs.io/en/latest/server-process.html#server-process-options).